### ISSUE TITLE
BBB 2.3 Beta 4 is not converting Office documents

### ISSUE THREAD

**Issue by makabar** [bigbluebutton/bigbluebutton#12065]
before upgrading from beta 3 i commented out 
`docker network remove bbb-libreoffice`
in 
_/var/lib/dpkg/info/bbb-libreoffice-docker.prerm_
[as suggested here](https://github.com/bigbluebutton/bigbluebutton/releases/tag/v2.3-beta-4)
While trying to upload a new Office document (tested with *.xslx and *.pptx) i get a endless loop.
`/usr/share/bbb-libreoffice-conversion/convert.sh`
runs @100% (also on small filesizes)

**Comment by PhMemmel** [comment]
Can confirm on my own installation as well as on test23.bigbluebutton.org

**Comment by makabar** [comment]
Solution:
replace 
/usr/share/bbb-libreoffice-conversion/convert-local.sh

[with this version](https://github.com/bigbluebutton/bigbluebutton/blob/develop/bbb-libreoffice/assets/convert-local.sh)

**Comment by ffdixon** [comment]
We can see the issue -- the install for `bbb-libreoffice-docker` is not escaping the `$` character.  Fixing now.

**Comment by makabar** [comment]
> We can see the issue -- the install for `bbb-libreoffice-docker` is not escaping the `$` character. Fixing now.

Thank You!!

**Comment by PhMemmel** [comment]
@makabar Can confirm your fix of the `convert-local.sh` ist doing well. Thank you!
@ffdixon thx for fixing this!

**Comment by erpsolns** [comment]
Sorry,  before upgrading from beta 3 i didnt commented out
docker network remove bbb-libreoffice.  I can upload pdf but not .pptx.
Now I changed /usr/share/bbb-libreoffice-conversion/convert-local.sh makabar  version - but still not converting and struck and gives error to delete.  What can I do now?

**Comment by erpsolns** [comment]
But my version was beta 3 before update

**Comment by makabar** [comment]
make a backup of /usr/share/bbb-web/WEB-INF/classes/bigbluebutton.properties if you changed there (WelcomeMessage or something like that)
comment out the line in /var/lib/dpkg/info/bbb-libreoffice-docker.prerm.
`apt update && apt dist-upgrade -y`
run your installscript again.
`wget -qO- https://ubuntu.bigbluebutton.org/bbb-install.sh | bash -s -- -v bionic-230 -s bbb.bbbexample.org -e yourmail@yourmail.com  -a -w`
**replace URL an MAIL!! before**
remove bbb-demo.
then this:
https://github.com/bigbluebutton/bigbluebutton/issues/12065#issuecomment-821969474

then restart bbb
`bbb-conf --restart`
worked for me.

**Comment by ffdixon** [comment]
We just released BigBlueButton 2.3-beta-5.  Can you upgrade and try it now and let us know if the conversion works.

**Comment by makabar** [comment]
just in time ;-)
i´ll going tu upgrade our 4th bbb server.
i´ll report soon!!

**Comment by makabar** [comment]
my upgrade pulled beta4 from repo :-(
so may report an other user.

**Comment by ffdixon** [comment]
Can you try upgrading again (you we just updated) packages-eu.bigbluebutton.org)

**Comment by makabar** [comment]
i´m on beta 5 now.
but i tried my hotfix above because i saw your reply too late :-(

**Comment by erpsolns** [comment]
Thanks for beta5 uploading pptx presentation works flawlessly.  Thank you ffdixon - you people are a gem

**Comment by erpsolns** [comment]
Hats off to BBB team - the swiftness to solve is unbelievable

**Comment by hostbbb** [comment]
Just ran bbb-install still pulling beta4 for me, with older bbb-config and libreoffice.
root@install:/home/sdame# dpkg -l | grep bbb- | sed 's/  [ ]*/ /g' | cut -d' ' -f1-3
ii bbb-apps-akka 2.3.0-70
ii bbb-config 1:2.3.0-60
ii bbb-demo 1:2.3.0-7
ii bbb-etherpad 1:2.3.0-47
ii bbb-freeswitch-core 2:2.3.0-12
ii bbb-freeswitch-sounds 1:1.6.7-1
ii bbb-fsesl-akka 2.3.0-41
ii bbb-html5 1:2.3.0-1617
ii bbb-libreoffice-docker 1:2.3.0-43
ii bbb-mkclean 1:0.8.7-1
ii bbb-playback 1:2.3.0-8
ii bbb-playback-presentation 1:2.3.0-9
ii bbb-record-core 1:2.3.0-30
ii bbb-web 1:2.3.0-70
ii bbb-webrtc-sfu 1:2.3.0-17

**Comment by ffdixon** [comment]
Can you post the output of

```
cat /etc/apt/sources.list.d/bigbluebutton.list
```

**Comment by ffdixon** [comment]
Ok, found the issue.  Please try updating again and let us know if you get build 45 of `bbb-libreoffice-docker`.

```
# dpkg -l | grep bbb-libreoffice-docker
ii  bbb-libreoffice-docker               1:2.3.0-45 
```

**Comment by antobinary** [comment]
This was addressed in 2.3-beta5. Please comment if you encounter further issues
